apps/staging-community/package.json

```diff
@@ -37,6 +37,7 @@
     "@grouparoo/postgres": "0.8.0-alpha.3",
     "@grouparoo/prometheus": "0.8.0-alpha.3",
     "@grouparoo/redshift": "0.8.0-alpha.3",
+    "@grouparoo/rollbar": "0.8.0-alpha.3",
```

Comment:
Please don't check this change in - we don't use rollbar (or newrelic for that matter) on our staging servers.